### PR TITLE
New Package: cherrytree-0.35.10

### PR DESCRIPTION
**Pull request by logenkain**

### CODE REVIEW

**Comment by logenkain** [comment]
Something wrong with this one?

File: srcpkgs/cherrytree/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'cherrytree'
+pkgname=cherrytree
+version=0.35.10
+revision=1
+noarch=yes
+build_style=python-module
+hostmakedepends="python-setuptools desktop-file-utils pygtksourceview"
```
Comment (steski) [inline srcpkgs/cherrytree/template]: `pygtksourceview` should be in `depends` instead

**Comment by asergi** [comment]
```diff
--- template.orig   2015-10-03 00:44:42.578736950 +0200
+++ template    2015-10-03 00:45:38.645036025 +0200
@@ -4,11 +4,21 @@
revision=1
noarch=yes
build_style=python-module
-hostmakedepends=”python-setuptools desktop-file-utils pygtksourceview”
-depends=”python python-dbus desktop-file-utils”
-short_desc=”Hierarchial note taking app featuring rich text and syntax highlighting”
+hostmakedepends=”python-setuptools desktop-file-utils”
+depends=”pygtksourceview python-dbus desktop-file-utils”
+pycompile_dirs=”usr/share/cherrytree/modules”
+short_desc=”Hierarchical note taking application”
maintainer=”Logen K logen@sudotask.com”
license=”GPL-3”
homepage=”http://www.giuspen.com/cherrytree/”
distfiles=”http://www.giuspen.com/software/pkgname−{version}.tar.xz”
checksum=34ed912074387a3a1e7b6b07d3b231b6a9c11165bbe52cbebf13b3b0ee46de12
+
+post_install() {
+	# don’t install useless egg
+	rm -rf {DESTDIR}/usr/lib/python2.7/site-packages  
+ 
+	# remove old mime registration files  
+	rm -rf ${DESTDIR}/usr/share/application-registry
+	rm -rf ${DESTDIR}/usr/share/mime-info
+}
```

**Comment by logenkain** [comment]
-hostmakedepends=”python-setuptools desktop-file-utils pygtksourceview”
+depends=”pygtksourceview python-dbus desktop-file-utils”

I can't get it to build on arm if I move pygtksourceview to depends line.

**Comment by steski** [comment]
Looks like `pygtksourceview` is broken on arm. Probably `nocross=yes` until it is fixed.

**Comment by logenkain** [comment]
On Sun, Oct 04, 2015 at 05:43:11PM -0700, steski wrote:
> Looks like `pygtksourceview` is broken on arm. Probably `nocross=yes` until it is fixed.
I suppose.  It is strange to me, however, that it would work being in
HOSTmakedepends... oh well.

**Comment by steski** [comment]
`pygtksourceview` is a runtime dependency therefore it must be defined in `depends` so it will be installed alongside `cherrytree`.

File: srcpkgs/cherrytree/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'cherrytree'
+pkgname=cherrytree
+version=0.35.10
+revision=1
+noarch=yes
+nocross=yes #pygtksourceview seems to be broken in arm
```
Comment (xtraeme) [inline srcpkgs/cherrytree/template]: You can now remove `nocross` and rebase/push to verify it can be cross compiled.

**Pull request merged**